compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/DecomposeAttention.cpp

```diff
@@ -46,9 +46,9 @@ static Value computeRowwiseReduction(Value a, Value output, Location loc,
   return genericOp.getResult(0);
 }
 
-static Value computePartialSoftmax(Value qkTranspose, Value currentMax,
-                                   Location loc, OpBuilder &builder,
-                                   SmallVectorImpl<Operation *> &ops) {
+Value computePartialSoftmax(Value qkTranspose, Value currentMax, Location loc,
+                            OpBuilder &builder,
+                            SmallVectorImpl<Operation *> &ops) {
```

Comment:
I think we prefer to have `static` keywords for all the local functions, even they are already under `anonymous` namespace. IREE follows mixed LLVM and Google style for compiler codebase. There is no convention on Google side, and there is a convention from LLVM style guide. The reason behind it is very solid, and we're following it in IREE codebase.

> The problem with anonymous namespaces is that they naturally want to encourage indentation of their body, and they reduce locality of reference: if you see a random function definition in a C++ file, it is easy to see if it is marked static, but seeing if it is in an anonymous namespace requires scanning a big chunk of the file.